test/test_processing_utils.py

```diff
@@ -131,5 +133,31 @@ def test_subclass_conversion(self):
             assert y.dtype == x.dtype
 
 
+def test_video_has_playable_codecs(test_file_dir):
```

Comment:
I added some files so that we can test positive and negative cases. I know increasing the repo size is a concern but I think testing the logic is worth adding the couple hundred kb